README.md

```diff
@@ -63,9 +63,11 @@ within Juttle. To do so, add the following to your `~/.juttle/config.json` file:
 }
 ```
 
-The `connection` key should point to your PostgreSQL instance.
+The `connection` key should point to your PostgreSQL instance. You may use the following documentation to create your connection string:
```

Comment:
That line now reads oddly because it's no longer linking to external documentation, how about just 
```
The format of the connection string is:
```